experimental/StandardFiniteFields/src/StandardFiniteFields.jl

```diff
@@ -23,10 +23,11 @@ function pop_largest_factor!(f::Fac{ZZRingElem})
 end
 
 # TODO : Should be fixed in Nemo
-function (k::Nemo.FpField)(a::Vector)
+function (k::FpField)(a::Vector)
   @assert isone(length(a))
   return k(a[1])
 end
+
 function (k::FqPolyRepField)(a::Vector)
```

Comment:
I guess this is similar to the above

Reply:
@mjrodgers  Also about `coords` methods a few lines down, where it says:
```
# TODO : Should be fixed in Hecke for prime fields
function coords(x::FinFieldElem)
  return absolute_coordinates(x)
end
function coords(x::PrimeFieldElem)
  return [x]
end
```
but there is no `coords` in Hecke, I think, so I am not sure that is about?

Reply:
There was an issue at the time that `absolute_coordinates` was not defined for fields of prime order (which meant they needed to be handled as a special case), so this was a workaround. I think now that the finite fields are unified, this can be removed; I'm looking into this.